src/app/campaign-list/campaign-list.module.ts

```diff
@@ -40,7 +44,12 @@ import { CampaignListService } from './campaign-list.service';
     CommonModule,
     PagingModule,
     RouterModule,
-    SharedModule
+    SharedModule,
```

Comment:
I had tried to import the material components as exported from SharedModule, but I kept getting errors about a date provider even though MatNativeDateModule was being imported. I moved back to importing material components in component closer to where we are using them. If you like Angular modules as much as I do, don't worry it's probably going to change again.